### PR TITLE
Add gwt 2.11 sdk feature

### PR DESCRIPTION
**Pull request by protoism**
GWT 2.11 at last.

### CODE REVIEW

File: features/com.gwtplugins.eclipse.sdkbundle.gwt211.feature/feature.properties [hunk 1]
```diff
@@ -0,0 +1,131 @@
+featureName  = GWT 2.11.0 SDK
+providerName = GWT Eclipse Plugin
+description = Downloads the GWT 2.11.0 SDK library 
+licenseURL = https://github.com/gwt-plugins/gwt-eclipse-plugin/blob/master/LICENSE.md
+
+# "license" property - text of the "Feature Update License"
+# should be plain text version of license agreement pointed to be "licenseURL"
+license=\
```
Comment (niloc132) [inline features/com.gwtplugins.eclipse.sdkbundle.gwt211.feature/feature.properties:8]: Shouldnt the URL, license be of the contents of GWT? If the user is asking to download GWT, shouldn't we be letting them know the license of it?

File: features/com.gwtplugins.eclipse.sdkbundle.gwt211.feature/feature.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+   <copyright>
+      Copyright 2015 to 2017
+   </copyright>
+
+   <license url="%licenseURL">
+      %license
+   </license>
+
+   <url>
+      <update label="GWT Eclipse Plugin Update Site" url="http://storage.googleapis.com/gwt-eclipse-plugin/v3/release"/>
```
Comment (niloc132) [inline features/com.gwtplugins.eclipse.sdkbundle.gwt211.feature/feature.xml:21]: This URL hasn't been correct in a while, I think we want to point to something like https://plugins.gwtproject.org/eclipse/gwt-eclipse-plugin/4.0.0/ instead?

**Review by niloc132 — DISMISSED** [review]
I briefly looked at past SDKs, and this seems to be consistent with them, but I'm not sure that this makes them right either...

Also did we miss GWT 2.10?

I approved because I don't know enough about this to say that we're wrong, but just wanted to check these things in case they mattered.

**Comment by protoism** [comment]
> I briefly looked at past SDKs, and this seems to be consistent with them, but I'm not sure that this makes them right either...
You mean the license?
Your observation makes definitely sense. 
Any link to the license (sorry I'm lazy)
> 
> Also did we miss GWT 2.10?

Adding GWT 2.10 is definitely possible.
In all honesty, for improved coherency, I'd remove 2.7, 2.8, 2.9.
It is possible and not that complicate to download the desired SDK.
Another solution is having a compound site, composed of:

GWT plugin
GWT SDK 2.7
GWT SDK 2.8
....
....
SDBG


> 
> I approved because I don't know enough about this to say that we're wrong, but just wanted to check these things in case they mattered.

**Comment by niloc132** [comment]
https://www.gwtproject.org/terms.html lists the license file, apache v2... 

We don't have a top-level file in the git repo (but we should) that shows the license. Here's the file that makes its way into the download zip: https://github.com/gwtproject/gwt/blob/main/distro-source/core/src/COPYING

to version that link, you could use https://github.com/gwtproject/gwt/blob/2.11.0/distro-source/core/src/COPYING (with the version instead of the branch).

--

Removing old versions is fine by me, 2.7 at least should probably be removed since I don't think it will run on the min Java version for eclipse any more (and possibly 2.8 too). Maybe in a follow-up PR?

I don't have strong opinions about how we should provide downloads, I'd vote that we don't worry about it too much for this release.

**Pull request merged**